central/role/default.go

```diff
@@ -57,12 +58,26 @@ var (
 	// AccessScopeIncludeAll gives access to all resources. It is checked by ID, as
 	// Rules cannot represent unrestricted scope.
 	AccessScopeIncludeAll = &storage.SimpleAccessScope{
-		Id:          EnsureValidAccessScopeID("unrestricted"),
+		Id:          getAccessScopeIncludeAllID(),
 		Name:        "Unrestricted",
 		Description: "Access to all clusters and namespaces",
 	}
 )
 
+func getAccessScopeExcludeAllID() string {
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
+		return "ffffffff-ffff-fff4-f5ff-fffffffffffe"
```

Comment:
nit:  Probably worth putting these as constants somewhere since we use this string here and in singleton.go